backend/tests/integration/common_utils/managers/chat.py

```diff
@@ -0,0 +1,144 @@
+import json
+
+import requests
+
+from danswer.file_store.models import FileDescriptor
+from danswer.llm.override_models import LLMOverride
+from danswer.llm.override_models import PromptOverride
+from danswer.one_shot_answer.models import DirectQARequest
+from danswer.one_shot_answer.models import ThreadMessage
+from danswer.search.models import RetrievalDetails
+from danswer.server.query_and_chat.models import ChatSessionCreationRequest
+from danswer.server.query_and_chat.models import CreateChatMessageRequest
+from tests.integration.common_utils.constants import API_SERVER_URL
+from tests.integration.common_utils.constants import GENERAL_HEADERS
+from tests.integration.common_utils.test_models import StreamedResponse
+from tests.integration.common_utils.test_models import TestChatMessage
+from tests.integration.common_utils.test_models import TestChatSession
+from tests.integration.common_utils.test_models import TestUser
+from tests.integration.tests.streaming_endpoints.utils import analyze_response
+
+
+class ChatSessionManager:
+    @staticmethod
```

Comment:
For object creation (and other overlapping functions), i think we should use the same name E.g. ChatSessionManager.create. The rest of the function names are good i think tho